.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "3akai-ux"]
 	path = 3akai-ux
   url = https://github.com/oaeproject/3akai-ux
+[submodule "node_modules/restjsdoc"]
```

Comment:
Including it in the codebase allows one to fiddle and experiment with both codebases at once without having to publish it every time there is a change.